### PR TITLE
Remove Boostrap

### PR DESCRIPTION
**Pull request by vsavkin**

### CODE REVIEW

File: modules/@angular/http/http.ts [hunk 1]
```diff
@@ -112,8 +109,10 @@ export {QueryEncoder, URLSearchParams} from './src/url_search_params';
  *   search: string = 'coreTeam=true';
  * }
  *
- * bootstrap(App, [HTTP_PROVIDERS, {provide: RequestOptions, useClass: MyOptions}])
- *   .catch(err => console.error(err));
+ * @NgModel({
```
Comment (robwormald) [inline modules/@angular/http/http.ts]: NgModel -> NgModule

File: modules/@angular/http/http.ts [hunk 2]
```diff
@@ -112,8 +109,10 @@ export {QueryEncoder, URLSearchParams} from './src/url_search_params';
...
+ *   providers: [HTTP_PROVIDERS, {provide: RequestOptions, useClass: MyOptions}]
```
Comment (robwormald) [inline modules/@angular/http/http.ts]: move this to import: [ HttpModule ]

**Comment by robwormald** [comment]
lgtm with the noted comments

**Comment by vikerman** [comment]
Travis failing

**Comment by vsavkin** [comment]
@vikerman fixed

**Pull request merged**